example_app_generator/no_active_record/spec/verify_no_fixture_setup_spec.rb

```diff
@@ -0,0 +1,22 @@
+# Pretend that ActiveRecord::Rails is defined and this doesn't blow up
+# with `config.use_active_record = false`.
+# Trick the other spec that checks that ActiveRecord is
+# *not* defined by wrapping it in RSpec::Rails namespace
+# so that it's reachable from RSpec::Rails::FixtureSupport.
+# NOTE: this has to be defined before requiring `rails_helper`.
+module RSpec
+  module Rails
+    module ActiveRecord
+      module TestFixtures
+      end
+    end
+  end
+end
```

Comment:
It's like that since the original problem was that `active_record` was defined, just wasn't used and DB wasn't set up.

I've double-checked that this test does blow up if `config.use_active_record = true` inside `no_active_record` app.

Reply:
I tried hard, but couldn't make [this example repo](https://github.com/JoostVanAverbeke/devise_remote_client/tree/usage_of_rspec-rails_3_3_2) work to reproduce the original issue. Mostly due to bundler 1/2 problem, and then dependencies hell with Rails version which I had to update to 5.2, and in the end with missing `rake-test`'s methods, probably due to version mismatch.